usage/gpio/README.md

```diff
@@ -1,87 +1,59 @@
+Two 5V pins are present on the board, as well as a number of ground pins (0V). The remaning pins are all general purpose 3V3 pins, meaning outputs are set to 3V3 and inputs are 3V3 tolerant.
 
-**Note**: Not **literally** anywhere, of course. You need things like access to the network, a network capable computing device, and electricity. Please do not write to us to point this out. :)
+## Outputs
 
-## How the GPIO pins work
+A GPIO pin designated as an output pin can be set to high (3V3) or low (0V).
 
-### Output
+### Inputs
 
-**WARNING**: If you follow the instructions, then messing about with the GPIO is safe and fun. Randomly plugging wires and power sources into your Pi, however, may kill it. Bad things can also happen if you try to connect things to your Pi that use a lot of power; LEDs are fine, motors are not. If you are worried about this, then you might want to consider using a breakout board such as the [Pibrella](http://pibrella.com/) until you are confident enough to use the GPIO directly.
+A GPIO pin designated as an input pin can be read as high (3V3) or low (0V). This is made easier with the use of internal pull-up or pull-down resistors. Pins GPIO2 and GPIO3 have fixed pull-up resistors, but other pins can configure this in software.
 
-Ignoring the Pi for a moment, one of the simplest electrical circuits that you can build is a battery connected to a light source and a switch (the resistor is there to protect the LED):
+## More
 
-![Simple circuit](images/simple-circuit.png)
+As well as simple input and output devices, the GPIO pins can be used with a variety of alternative functions, some are available on all pins, others on specific pins.
 
-When we use a GPIO pin as an output, the Raspberry Pi replaces **both the switch and the battery** in the above diagram. Each pin can turn on or off, or go HIGH or LOW in computing terms. When the pin is HIGH it outputs 3.3 volts (3v3); when the pin is LOW it is off.
+- PWM (pulse-width modulation)
+    - Software PWM available on all pins
+    - Hardware PWM available on GPIO12, GPIO13 and GPIO18
+- SPI
+    - SPI0: MOSI (GPIO10); MISO (GPIO9); SCLK (GPIO10); CE0 (GPIO8), CE1 (GPIO7)
+    - SPI1: MOSI (GPIO20); MISO (GPIO19); SCLK (GPIO21); CE0 (GPIO18); CE1 (GPIO17); CE2 (GPIO16)
+- I2C
+    - Data: (GPIO2); Clock (GPIO3)
+    - EEPROM Data: (GPIO0); EEPROM Clock (GPIO1)
```

Comment:
Refers to GPIO0 and GPIO1 which are not on the diagram at top of page, which some might find confusing.

Reply:
Added a note at the top